openrpc.json

```diff
@@ -283,6 +283,68 @@
         }
       ]
     },
+    {
+      "name": "wallet_revokePermissions",
+      "tags": [
+        {
+          "$ref": "#/components/tags/MetaMask"
+        }
+      ],
+      "summary": "Revoke wallet permissions.",
+      "description": "Revoke previously granted permissions for a specific dapp identified by origin (invoker)",
+      "paramStructure": "by-name",
+      "params": [
+        {
+          "name": "permission",
+          "required": false,
+          "schema": {
+            "type": "object",
+            "properties": {
+              "target": {
+                "type": "string",
+                "description": "The target name of the permission",
+                "enum": ["eth_accounts"]
```

Comment:
Is it possible to make the spec open-ended so it doesn't need to be updated whenever a new permission is added?